File: src/main/java/popa/catalin/lab3/pieces/Peon.java
package popa.catalin.lab3.pieces;

import popa.catalin.lab3.Position;

public class Peon extends ChessPiece {
    public Peon(int vertPos, int horizPos, boolean isAlive, ChessPieceColor color) {
        super(new Position(vertPos, horizPos), isAlive, color);
    }

}

File: LabsSimonaBurlacu232/Lab3/ChessGame/ChessGame/Pieces/Peon.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ChessGame.Pieces
{
    public class Peon : ChessPiece
    {
        public Peon(Position position, bool isAlive, ChessPieceColor color) : base(position, isAlive, color)
        { }
        
    }
}
